### ISSUE TITLE
vertex normalization for general SC sets

### ISSUE THREAD

**Issue by mdsumner** [hypertidy/anglr#79]
DEL won't work here because the edges got denormalized by copy_down (?)

```R
DEL(copy_down(SC(cont_tas), "ELEVATION"))
[1] "edge_"       "object_"     "edge_vertex" "z_"          "x_"          "y_"          "vertex_"    
dropping untriangulatable objects
 Error in RTriangle::pslg(P = as.matrix(x[["vertex"]][c("x_", "y_")]),  : 
  Duplicated vertices in P. 
```

**Comment by mdsumner** [comment]
as per comment in DEL.SC

```
  ## we need a pfft::edge_triangle_map
  ## https://github.com/hypertidy/silicate/issues/62#issuecomment-372898877
```

**Comment by mdsumner** [comment]
This can't really work because the copy down has split the mesh, so really each separate mesh should have DEL applied to individually. What we can now do though (having fixed ff38499da678e8b473e5f9c3d2dc4d8205cb9389) is

*copy the elevation after DEL mesh*: 

```R
(x <- copy_down(DEL(SC(cont_tas)), "ELEVATION"))
dropping untriangulatable objects
class       : DEL
type        : Primitive
vertices    : 771 (3-space)
primitives  : 257 (2-space)
crs         : +proj=utm +zone=55 +south +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs

plot3d(x)
```


![image](https://user-images.githubusercontent.com/4107631/81997728-fbf94e00-9693-11ea-848e-d921af0b9a2d.png)
